### PR TITLE
Test all the infra stuff

### PR DESCRIPTION
**Pull request by mattgen88**
Holding until I get a chance to thoroughly test the changes I've made. 

Man we need API tests

### CODE REVIEW

File: infrastructure/authors/authors.go [hunk 1]
```diff
@@ -8,18 +8,24 @@ import (
...
 func New(db *gorm.DB) *Repository {
 	return &Repository{db: db}
 }
 
+// Repository is used for querying and creating authors
 type Repository struct {
 	db *gorm.DB
 }
 
+// GetByID returns an author from the database given an ID
```
Comment (Sigafoos) [inline infrastructure/authors/authors.go:27]: To be pedantic, they both need periods.

File: infrastructure/books/books.go [hunk 2]
```diff
@@ -8,26 +8,37 @@ import (
...
 func New(db *gorm.DB) *Repository {
 	return &Repository{db: db}
 }
 
+// Repository is used for querying and creating books
 type Repository struct {
 	db *gorm.DB
 }
 
+// GetByID returns a book from the database given an ID
```
Comment (Sigafoos) [inline infrastructure/books/books.go:29]: ditto

File: infrastructure/registrationkeys/registrationkeys.go [hunk 3]
```diff
@@ -9,20 +9,28 @@ import (
...
 func New(db *gorm.DB) *Repository {
 	return &Repository{db: db}
 }
 
+// Repository is used for querying and creating registration keys
 type Repository struct {
 	db *gorm.DB
 }
 
+// GetByUsername returns a registration key given a user's username
```
Comment (Sigafoos) [inline infrastructure/registrationkeys/registrationkeys.go:30]: periods! periods everywhere!

File: infrastructure/authors/authors.go (hunk 1 above)
Comment (mattgen88) [reply infrastructure/authors/authors.go:27]: I'm sorry, did you forget this is golang? We don't end our lines with punctuation. Ever. Period.

Comment (Sigafoos) [reply infrastructure/authors/authors.go:27]: > Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared.

😇

**Comment by Sigafoos** [comment]
@mattgen88 is this still a WIP?

**Pull request merged**